src/foam/nanos/notification/NotificationMessageModal.js

```diff
@@ -68,7 +76,7 @@ foam.CLASS({
       .start(this.Rows)
         .addClass(this.myClass('container'))
         .start().addClass('p-bold').add(this.MESSAGE_MSG).end()
-        .start().addClass(this.myClass('message')).add(this.data.body$).end()
+        .start().addClass(this.myClass('message')).add(this.description$).end()
```

Comment:
This line is not working. No errors, just nothing rendered.  The expression is not called.

Reply:
> This line is not working. No errors, just nothing rendered. The expression is not called.

Well, apparently just not working for me. works for others.